doc/sign-package.md

```diff
@@ -79,44 +81,41 @@ Included is a sample JSON configuration where the elements will be described nex
 
 First we have the global settings for Mystikos.
 
----
-
-Setting | Description
--|-
-version | Mystikos configuration version number. If the schema version is changed within Mystikos this version ties this configuration to a specific schema version
+Setting | Type | Description
+-|-|-
+version | `string` | Mystikos configuration version number. If the schema version is changed within Mystikos this version ties this configuration to a specific schema version. Current version is `"0.1"`
 
 ---
 
 Next we have settings specific to configuring the SGX enclave itself.
 
----
-
-Setting | Description
--|-
-Debug | Enable debugging within the SGX enclave, turn off for release builds
-ProductID | The product ID of your application. This is an integer value
-SecurityVersion | Security version of your application. This is an integer value.
+Setting | Type | Description
+-|-|-
+Debug | `boolean \| int` | Enable debugging within the SGX enclave, turn off for release builds
```

Comment:
Can we do "Debug: true"? If not then take out `boolean`.

Reply:
We should be able to use boolean 
https://github.com/deislabs/mystikos/blob/main/tools/myst/config.c#L130